<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationEditDialog</class>
 <widget class="QDialog" name="OperationEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>611</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="operationNamelineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Return type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="returnTypeComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Visibility:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="privateRadioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Private</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">visibilityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="publicRadioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Public</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">visibilityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="protectedRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Protected</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">visibilityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="packageRadioButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Package</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">visibilityButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QTableWidget" name="variablesTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variables:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addVariablePushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addConditionPushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Conditions:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteConditionPushButon">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="conditionsTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OperationEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OperationEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="visibilityButtonGroup"/>
 </buttongroups>
</ui>
